src/videoCache.js

```diff
@@ -162,6 +170,36 @@ export function getCacheUrl(id) {
   return `${config.getConfig('cache.url')}?uuid=${id}`;
 }
 
+export const storeLocally = (bid) => {
+  const vastValue = getVastValue(bid);
+  const dataUri = 'data:text/xml;base64,' + btoa(vastValue);
+  bid.vastUrl = dataUri;
+  //@todo: think of wrapping it with [if (adServer)]
+  vastsLocalCache.set(getLocalCacheBidId(bid), dataUri);
+};
+
+export async function getLocalCachedBidWithGam(adTagUrl) {
```

Comment:
This should be in the dfp module. 

IMO the GAM flow in general needs to change, instead of calling dfp's buildVideoUrl and passing the URL to the player it should be something like getVast and pass the XML to the player, which should work regardless of whether one is using local cache, and should also be the same thing that's called by the video module (regardless of the type of cache in use).

It might help to update / add a non-video module example to flesh out how the new API should look.